app/test_main.py

```diff
@@ -1,3 +1,54 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age",
+    [
+        pytest.param(
+            5,
+            6,
+            [0, 0],
```

Comment:
Placing arguments in several lines decreases the readability, place arguments in one line instead, please
```suggestion
            5, 6, [0, 0],
```